### PR TITLE
Replace D2k 8-bit custom tiles with 16-bit equivalents

### PR DESCRIPTION
**Pull request by Porenutak**
Dependencies:  #21240
Tested on devtest-20231206 version.
This PR replace .tmp custom tiles with R16 equivalents and add some new from #21221
Know issue:
I was unable to use PngSheet because utility convert it back to 8-bit. Instead I used standart R16 tileset format. But because Tileset editor require exact resolution (640x 1280) there is lots of unused frames there.
Tileset:
customtiles

Im planning to fill free frames in later PRs.

### CODE REVIEW

**Review by PunkPun — COMMENTED** [review]
Overall LGTM, need a rebase

**Comment by PunkPun** [comment]
Did you accidentally push balance changes into this PR?

**Comment by pchote** [comment]
Can we please move the customtiles.r16 from the `tex` subdirectory to `bits`?
The `tex` dir can then be removed from mod.yaml.

**Comment by pchote** [comment]
`TmpRA` can also be removed from `SpriteFormats` in mod.yaml.

**Comment by Porenutak** [comment]
Sorry, I fuck up my rebase again :( . 
 customtiles.r16 moved to bits folder. 
TmpRA was removed from mod.yaml (hope no custom map used any :D )

I check all custom tiles once more and fixed all pixel size errors. But should be check also by someone else.

**Comment by Porenutak** [comment]
Is this PR working now? because it start crashing after I force push.

File: .gitignore [hunk 1]
```diff
@@ -54,3 +54,5 @@ update.log
 
 # IntelliJ files
 .idea
+mods/d2k/maps/tiletest/map.bin
+mods/d2k/maps/tiletest/map.png
```
Comment (PunkPun) [inline .gitignore]: please revert these changes

**Review by PunkPun — COMMENTED** [review]
it seems to be working for me.

File: .gitignore (hunk 1 above)
Comment (Porenutak) [reply .gitignore]: deleted

**Review by PunkPun — APPROVED** [review]
LGTM, couldn't find any regressions and @pchote gave a +2 on discord

**Comment by PunkPun** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/ea7653e6d6f9f4ad2da48ef2b474d8d452e492ed...d2e01d157a224ab6b4f92a038868738e97feab85)
